### PR TITLE
emscripten 1.30.2

### PR DESCRIPTION
**Pull request by mjbshaw**

### CODE REVIEW

File: Library/Formula/emscripten.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Emscripten < Formula
   homepage "https://kripken.github.io/emscripten-site/"
-  url "https://github.com/kripken/emscripten/archive/1.29.11.tar.gz"
-  sha1 "d926f406486e3c3a94a1fe58783a91fd6975bc9c"
+  url "https://github.com/kripken/emscripten/archive/1.30.2.tar.gz"
+  sha1 "20336fd538adb942b95beadec1fdabba55a08767"
```
Comment (DomT4) [inline Library/Formula/emscripten.rb]: If you're feeling super-kind, you could change all the ` sha1 ` to ` sha256 `.

Comment (mjbshaw) [inline Library/Formula/emscripten.rb]: Done!

Comment (DomT4) [inline Library/Formula/emscripten.rb]: Thanks!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**